src/fragments/lib/storage/js/download.mdx

```diff
@@ -137,6 +137,42 @@ You can use `expires` option to limit the availability of your URLs. This config
 await Storage.get('filename.txt', { expires: 60 });
 ```
 
+### Example
+
+Below is an example of a React component with a Download link to a file:
+
+```javascript
```

Comment:
Should this be titled 'React' instead?

Reply:
I just added an Example section in which this or other examples could live.  If we add additional examples, we could use subheaders or paragraphs for those

Reply:
Makes sense, thank you!